cmd/createFgaModel.go

```diff
@@ -79,6 +90,49 @@ func createModel(apiUrl, apiToken, storeId string) {
 	}
 
 	fmt.Printf("Created model: %s\n", modelId)
+
+	if err := storeValuesK8sConfigMap(ctx, kubeconfig, k8sConfigMap, storeId, modelId); err != nil {
+		panic(err)
+	}
+}
+
+func storeValuesK8sConfigMap(ctx context.Context, kubeconfig, configMap, storeID, modelID string) error {
+	if configMap == "" {
+		return nil
+	}
+
+	k8sClient, err := k8s.NewCoreV1Client(kubeconfig)
+
+	if err != nil {
+		return err
+	}
+
+	cmNamespace := ""
+	cmName := ""
+
+	if k8sResource := strings.Split(configMap, "/"); len(k8sResource) == 2 {
```

Comment:
Nitpick(non-blocking):

Maybe this is better and consistent with other error handling?

```golang
parts := strings.Split(configMap, "/")
if len(parts) != 2 {
	return fmt.Errorf("invalid format for configmap resource %s: expected namespace/name", configMap)
}
namespace, name := parts[0], parts[1]
```